### PR TITLE
Added displayLights menu

### PR DESCRIPTION
**Pull request by aardschok**
__Additions__
- Implemented lighting menu, pick type of display lighting for playblast

__Changes__
- Restructured UI for Viewport Plugin

### CODE REVIEW

File: capture_gui/plugins/viewportplugin.py [hunk 1]
```diff
@@ -17,33 +16,53 @@ class ViewportPlugin(capture_gui.plugin.Plugin):
     def __init__(self, parent=None):
         super(ViewportPlugin, self).__init__(parent=parent)
 
+        # set inherited attributes
+        self.setObjectName(self.label)
+
+        # cistom atttributes
         self.show_type_actions = list()
-        self.show_types = lib.get_show_object_types()
+        self.display_lights_acitons = list()
```
Comment (BigRoy) [inline capture_gui/plugins/viewportplugin.py]: Typo in `actions`

**Comment by BigRoy** [comment]
Should this also implement _two sided lighting_ and _shadows_ toggle? :) 
Again, that is also a question related to #53 discussion.

**Comment by aardschok** [comment]
"Fixed issue with two sided lighting and shadows being checkboxes" should be:
"Fixed issue with two sided lighting and shadows not being checkboxes". They needed to be accessible regardless of the type of lighting chooses, see Lighting menu in Maya viewport

**Comment by aardschok** [comment]
Fix #52

**Pull request merged**